### PR TITLE
Adding nearest sampling

### PR DESCRIPTION
**Pull request by shruthirai**

### CODE REVIEW

File: react/src/lib/components/DeckGLMap/layers/colormap/colormapLayer.ts [hunk 1]
```diff
@@ -103,6 +111,9 @@ export interface ColormapLayerProps extends BitmapLayerProps {
 
     // check for logarithmic values
     isLog?: boolean;
+
+    // // check for nearest interpolation
+    isNearest?: boolean;
```
Comment (hkfb) [inline react/src/lib/components/DeckGLMap/layers/colormap/colormapLayer.ts:116]: I don't think we should need these settings. The `colorMapFunction` should cover interpolation as well as breakpoints.

**Review by hkfb — CHANGES_REQUESTED** [review]

**Comment by shruthirai** [comment]
Breakpoints will be handled with this task https://github.com/equinor/webviz-subsurface-components/issues/1297

**Comment by shruthirai** [comment]
@hkfb This is ready for review

**Review by hkfb — APPROVED** [review]
We should stop using the `ColorMapLayer` and use the `MapLayer` instead for all relevant stories. Please move this story to `react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx` (in a separate PR if you like)

**Pull request merged**